packages/component/.eslintrc.yml

```diff
@@ -5,3 +5,6 @@ extends:
 # This package is expected to run inside a web browser.
 env:
   browser: true
+
+rules:
```

Comment:
Maybe put at root `.eslintrc.typescript.yml`?

Reply:
I guess you're referring to `.eslintrc.production.yml`